.laminas-ci.json

```diff
@@ -6,6 +6,7 @@
     ],
     "ignore_php_platform_requirements": {
         "8.0": true,
-        "8.1": true
+        "8.1": true,
```

Comment:
Do we still need this for 8.1?

Reply:
We need it for ^8 - because of `laminas-console` at least